### ISSUE TITLE
Alert Dialog, page jumps when open

### ISSUE THREAD

**Issue by jasongitmail** [huntabyte/shadcn-svelte#143]
### Current Behavior

Clicking the alert dialog causes background page to shift. It's the scroll bar disappearing.

### Expected Behavior

Page should not jump.

### Steps To Reproduce

1. Click Alert Dialog.

### More Information

This is more site related than component related, but mentioning where I saw it.

This can be fixed with:

```
html {
    /* Prevent UI jumps between pages with and without a scroll bar */
    overflow-y: scroll;
}
```

**Comment by huntabyte** [comment]
Thanks @jasongitmail ! Will implement this soon!

**Comment by huntabyte** [comment]
Closed by #150
